frontend/components/card/CardDangerZone.vue

```diff
@@ -3,50 +3,44 @@
     class="px-5 py-5 border bg-light-act-red/30 dark:bg-dark-act-red/30 border-light-act-red dark:border-dark-act-red card-style-base text-light-text dark:text-dark-text"
   >
     <div class="flex flex-col gap-5">
-      <h3
-        class="font-bold text-left text-light-act-red dark:text-dark-act-red responsive-h3 font-display"
-      >
+      <h3 class="font-bold text-left text-light-act-red dark:text-dark-act-red responsive-h3 font-display">
         {{ $t("components.card-danger-zone.header") }}
       </h3>
-      <p class="">{{ description }}</p>
```

Comment:
Where did all these come from?? 🤦‍♂️😅

Reply:
You mean the empty `class` attributes?

Reply:
Yes :) No stress on this, just a bit of light venting 😊